<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FractalDimensionWindow</class>
 <widget class="QWidget" name="FractalDimensionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>932</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FractalDimensionWindow</string>
  </property>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Max useful box:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_minUsefulBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_computeFractalDim">
   <property name="geometry">
    <rect>
     <x>413</x>
     <y>30</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose and compute...</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_maxUsefulBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_fractalDimension">
   <property name="geometry">
    <rect>
     <x>733</x>
     <y>30</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_fractalDim">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>850</width>
     <height>850</height>
    </rect>
   </property>
   <property name="renderHints">
    <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>613</x>
     <y>30</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Fractal Dimension:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Min useful box:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_computeFractalDim</sender>
   <signal>clicked()</signal>
   <receiver>FractalDimensionWindow</receiver>
   <slot>slotComputeFractalDimension()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>580</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotComputeFractalDimension()</slot>
 </slots>
</ui>
